solidity/contracts/libraries/staking/MinimumStakeSchedule.sol

```diff
@@ -4,17 +4,29 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 
 /// @notice MinimumStakeSchedule defines the minimum stake parametrization and
 /// schedule. It starts with a minimum stake of 100k KEEP. Over the following
-/// 2 years, the minimum stake is lowered periodically using a uniform stepwise
-/// function, eventually ending at 10k.
+/// 2 years, starting from the moment KEEP token has been deployed, the minimum
+/// stake is lowered periodically using a uniform stepwise function, eventually
+/// ending at 10k.
 library MinimumStakeSchedule {
     using SafeMath for uint256;
 
+    // Apr-28-2020 02:52:46 AM UTC when KEEP token has been deployed
```

Comment:
Let's make this yyyy-mm-dd and drop the AM at some point. Which is to say, let's stick to ISO8601 for dates where possible.